sklearn/linear_model/_perceptron.py

```diff
@@ -9,6 +9,16 @@
 class Perceptron(BaseSGDClassifier):
     """Linear perceptron classifier.
 
+    The implementation is a wrapper around :class:`~sklearn.linear_model.SGDClassifier`
+    by fixing the `loss` and `learning_rate` parameters as::
+
+        SGDClassifier(loss="perceptron", learning_rate="constant")
+
+    Other available parameters are described below and are forwarded to
+    :class:`~sklearn.linear_model.SGDClassifier`. Refer to
+    :ref:`mathematical section <sgd_mathematical_formulation>` of the SGD procedure for
+    more details.
+
     Read more in the :ref:`User Guide <perceptron>`.
```

Comment:
Thanks for the PR @glemaitre, it is certainly clearer now.

I just think that having two references to different places in the user guide ([sgd_mathematical_formulation](https://scikit-learn.org/stable/modules/sgd.html#mathematical-formulation) and later [perceptron](https://scikit-learn.org/stable/modules/linear_model.html#perceptron)) may increase the number of click to access relevant information. Can we instead keep only the [perceptron](https://scikit-learn.org/stable/modules/linear_model.html#perceptron) reference? The we can either add the respective loss formula there or link to [sgd_mathematical_formulation](https://scikit-learn.org/stable/modules/sgd.html#mathematical-formulation) from there. WDYT?

Reply:
I moved the details of the math link in the user guide.